force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -322,17 +376,17 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer getSalary(String employeeName) {
         // Initialize a Map
-
         // Add key-value pairs to the Map
+        Map<String, Integer> employeesDB = new Map<String, Integer>{'John Doe' => 50000, 'Jane Smith' => 60000, 'Sam Brown' => 55000, 'Alice Johnson' => 65000};
+
         // add John Doe => 50000
         // add Jane Smith => 60000
         // add Sam Brown => 55000
         // add Alice Johnson => 65000
-
         // Get the salary of the employee
-
+        Integer salary = employeesDB.get(employeeName);
         // Return the salary of the employee, or null if the employee is not found in the Map
-        return null; // Replace null with the variable you used to store the result
+        return salary; 
```

Comment:
Excellent. The only change I would make is to skip the variable assignment and return directly from line 387.